### PR TITLE
Add VanPy and VanJS to website

### PR DESCRIPTION
**Pull request by Qard**
Updating the website to include VanPy and VanJS.

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -99,19 +109,19 @@ class Index extends React.Component {
             <ul className="statistics">
               <li className="style1">
                 <span className="icon fa-code-fork"></span>
-                <strong>4</strong> meetups
+                <strong>6</strong> meetups
               </li>
               <li className="style3">
                 <span className="icon fa-signal"></span>
-                <strong>7,931</strong> members
+                <strong>12,000+</strong> members
```
Comment (caffodian) [inline src/pages/index.js:116]: Don't actually do this if you've stepped away or something,

but we could just make `NUM_MEMBERS` a variable if you want the two to match easily ;)

File: src/pages/index.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ class Index extends React.Component {
 
           <section id="first" className="main special">
             <header className="major">
-              <h2>founding events</h2>
+              <h2>our events</h2>
```
Comment (caffodian) [inline src/pages/index.js:62]: holy github diff

**Review by caffodian — APPROVED** [review]

File: src/pages/index.js (hunk 2 above)
Comment (keywordnew) [reply src/pages/index.js:62]: 😄

**Review by keywordnew — APPROVED** [review]
LGTM! Thank you for this PR. Very much needed! 🎉

**Comment by keywordnew** [comment]
Merging, folks!

**Pull request merged**